python/lsst/cp/pipe/ptc.py

```diff
@@ -978,6 +979,11 @@ def errFunc(p, x, y):
                 dataset.nonLinearityResiduals[ampName] = np.nan
                 dataset.fractionalNonLinearityResiduals[ampName] = np.nan
                 dataset.coefficientLinearizeSquared[ampName] = np.nan
+                dataset.ptcFitPars[ampName] = np.nan
+                dataset.ptcFitParsError[ampName] = np.nan
+                dataset.ptcFitReducedChiSquared[ampName] = np.nan
+                dataset.coefficientsLinearizePolynomial[ampName] = [np.nan for _ in range(lenNonLinPars)]
```

Comment:
`[np.nan] * lenNonLinPars`

Reply:
Also though, is this any more necessary than the other nan-lists which you removed?